examples/install.yaml

```diff
@@ -6,14 +6,14 @@ metadata:
   labels:
     app: crossplane-provider-terraform
 spec:
-  image: crossplane/provider-terraform-controller:v0.2.0
+  image: crossplanecontrib/provider-terraform-controller:v0.5.0
```

Comment:
Actually this line can be removed after we switched to new 'fat' combined image where we have both controller and CRDs

Reply:
Nice, should I keep the `args: ["-d"]`?

Reply:
yes, so it will still keep the verbose logging